### PR TITLE
[DE-Migration] LPS-125457 replace with site default locale

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-125457

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Review by jeyvison — CHANGES_REQUESTED** [review]
hi @marcosapmf. Please, see my comment! Thanks ^^

**Comment by oliveiraaraujo** [comment]
LGTM!

File: modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/strategy/DefaultMapToDDMFormValuesConverterStrategy.java [hunk 1]
```diff
@@ -141,9 +142,19 @@ public void setDDMFormFieldValues(
...
+				Locale siteDefaultLocale = LocaleUtil.getSiteDefault();
+
 				list = (List<Object>)localizedValues.get(
 					LanguageUtil.getLanguageId(
-						(Locale)GetterUtil.getObject(locale, defaultLocale)));
+						(Locale)GetterUtil.getObject(
+							locale, siteDefaultLocale)));
+
+				if (!siteDefaultLocale.equals(defaultLocale) &&
+					((list == null) || list.isEmpty())) {
```
[the remark this answers is not in the document]
Comment (marcosapmf) [reply modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/strategy/DefaultMapToDDMFormValuesConverterStrategy.java]: Hey @jeyvison, I just forced a push for this pull with this change. Can you please take a look at it?

Thanks!

**Comment by marcosapmf** [comment]
ci:stop

**Comment by marcosapmf** [comment]
ci:test:sf

**Review by jeyvison — APPROVED** [review]
LGTM!

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:forward

**Pull request closed without merging**